packages/forms/src/form_builder.ts

```diff
@@ -315,13 +315,16 @@ export class FormBuilder {
  *
  * @publicApi
  */
-export interface NonNullableFormBuilder {
+@Injectable(
+    {providedIn: ReactiveFormsModule, useFactory: () => inject(FormBuilder).nonNullable},
+    )
```

Comment:
Note: the comma is in the wrong place (it should be right after the `nonNullable` field):
```suggestion
@Injectable({
  providedIn: ReactiveFormsModule,
  useFactory: () => inject(FormBuilder).nonNullable,
})
```